src/annotator/test/sidebar-test.js

```diff
@@ -234,6 +234,28 @@ describe('Sidebar', () => {
         assert.called(target);
       }));
 
+    describe('on "showNotebook" event', () => {
+      it('hides itself and republishes the event', () => {
+        const sidebar = createSidebar();
+        sinon.stub(sidebar, 'publish');
```

Comment:
I think would be better/more realistic here to create another `Delegator` which subscribes to the events with a stub as the callback rather than stubbing `publish`.

Reply:
This pattern is copied from elsewhere in these tests. I agree with you, overall. I went in to start replacing this with a `Delegator` instance but it turned out to be more complex than I'd hoped; may I punt this until the next round of changes? I already have a small list of things I'd like to change.

Reply:
OK, let's punt on this for now.